Zend/zend_language_parser.y

```diff
@@ -1561,6 +1564,24 @@ isset_variable:
 		expr { $$ = zend_ast_create(ZEND_AST_ISSET, $1); }
 ;
 
+clone:
+		T_CLONE expr { $$ = zend_ast_create(ZEND_AST_CLONE, $2, NULL); }
+	|	T_CLONE expr T_WITH property_initializer_list { $$ = zend_ast_create(ZEND_AST_CLONE, $2, $4); }
+
+property_initializer_list:
```

Comment:
Is it the intention to use this new sort of `property_initialiser_list` in further new features too? If not, then I would suggest prefixing it (and the following two definitions with `clone`_.